pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -217,15 +217,24 @@ func (p *perfCounterNodeStatsClient) getCPUUsageNanoCores() uint64 {
 }
 
 func getSystemUUID() (string, error) {
-	result, err := exec.Command("wmic", "csproduct", "get", "UUID").Output()
+	// Use PowerShell cmdlet "Get-CimInstance" instead of "wmic csproduct get
+	// uuid" for the following reasons:
+	// 1. The WMI command-line tool (Wmic) is deprecated. Use PowerShell cmdlets
+	// instead.
+	// https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/hh831568(v=ws.11)#wmi-providers
+	// 2. The PowerShell cmdlet "Get-CimObject" was removed from PowerShell 6
+	// and are no longer supported.
+	// https://github.com/MicrosoftDocs/PowerShell-Docs/issues/5156#issuecomment-558253083
+	command := "(Get-CimInstance -Class Win32_ComputerSystemProduct).UUID"
+	args := []string{"-NoProfile", "-NonInteractive", "-ExecutionPolicy", "Restricted", "-Command", command}
+	result, err := exec.Command("powershell.exe", args...).CombinedOutput()
 	if err != nil {
-		return "", err
+		return "", fmt.Errorf("unable to get system uuid, reason: %q, detail: %q", err, result)
```

Comment:
is there any reason to use `%q` for these? %v for `err` and %s for `result` might be better.